docs/azure-sql-db.md

```diff
@@ -0,0 +1,163 @@
+# Azure SQL Database Service
+
+[Azure SQL Database](https://azure.microsoft.com/en-us/documentation/articles/sql-database-technical-overview/) is a relational database service in the cloud based on the market-leading Microsoft SQL Server engine, with mission-critical capabilities.
+
+## Create an Azure SQL Database service
+
+1. Get the service name and plans
+
+  ```
+  cf marketplace
+  ```
+
+  Sample output:
+
+  ```
+  service                       plans                     description
+  azuresqldb                    basic, StandardS0         Azure SQL Database Service
+  ```
+
+  If you can not find the service name, please use the following command to make the plans public.
+
+  ```
+  cf enable-service-access azuresqldb
+  ```
+
+2. Create a service instance
+
+  Configuration parameters are supported with the provision request. These parameters are passed in a valid JSON object containing configuration parameters, provided either in-line or in a file.
+
+  ```
+  cf create-service azuresqldb $service_plan $service_instance_name -c $path_to_parameters
+  ```
+
+  Supported configuration parameters:
+
+  ```
+{
+    "resourceGroup": "<resource-group>",
+    "sqlServerName": "<sql-server-name>",
+    "sqlServerCreateIfNotExist": true | false,
+    "sqlServerParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "administratorLogin": "<sql-server-admin-name>",
+            "administratorLoginPassword": "<sql-server-admin-password>"
+        }
+    },
+    "sqldbName": "<sql-database-name>",
+    "sqldbParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "collation": "SQL_Latin1_General_CP1_CI_AS | <or-other-valid-sqldb-collation>"
+        }
+    }
+}
+  ```
+
+  For example:
+
+  ```
+  cf create-service azuresqldb basic mysqldb -c /tmp/config.json
+  ```
+
+  ```
+  {
+    "resourceGroup": "sqldbResourceGroup",
+    "sqlServerName": "sqlServerA",
```

Comment:
`Servername cannot be empty or null. It can only be made up of lowercase letters 'a'-'z', the numbers 0-9 and the hyphen. The hyphen may not lead or trail in the name.`

1. `sqlServerA` is invalid.
2. Let's note it down in the doc to remind users.